src/SignInPage.tsx

```diff
@@ -46,10 +47,18 @@ const useStyles = makeStyles(theme => ({
 }))
 
 type SignInPageProps = {
+  isLoadingLoginWithUsernameAndPassword: UseLoginReturn['isLoadingLoginWithUsernameAndPassword']
+  submitUsernameAndPassword: UseLoginReturn['submitUsernameAndPassword']
+  errorMessageLoginWithUsernameAndPassword: UseLoginReturn['errorMessageLoginWithUsernameAndPassword']
```

Comment:
Is there a cleaner way to inherit these fields from UseLoginReturn?

Reply:
I've refactored to pass down an object with these props, rather than passing each down separately. Happy to update if there's a better approach!